CoherentTasks/CoherentTasks/MaxMinArraySum.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CoherentTasks
+{
+    class MaxMinArraySum
+    {
+        public static void Execute()
+        {
+            Console.Write("Please enter your array with space between numbers: ");
+            string inputArray = Console.ReadLine();
+
+            int answer = ArrayMaxMinSum(inputArray);
+
+            Console.WriteLine($"This is your array: {inputArray}");
+            Console.WriteLine($"The sum of this array range starting from first highest number until lowest number after is {answer}");
+        }
+
```

Comment:
Better to transfer the int[], not string from console. Single responsibility, one method has to transform string to array, and  ArrayMaxMinSum(string array) has to be  ArrayMaxMinSum(int[])